src/SWP/Bundle/ContentBundle/Loader/ArticleLoader.php

```diff
@@ -125,7 +126,34 @@ public function load($type, $parameters, $responseType = LoaderInterface::SINGLE
                 ));
 
                 if ($route) {
-                    $articles = $this->dm->getReferrers($route, null, null, null, 'SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\Article');
+                    $node = $this->dm->getNodeForDocument($route);
+                    $identifier = $node->getIdentifier();
+
+                    $queryStr = 'SELECT * FROM [nt:unstructured] as S ';
+                    $queryStr .= "WHERE S.phpcr:class='SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\Article' ";
+                    $queryStr .= "AND S.route=$identifier";
+
+                    if (isset($parameters['order'])) {
+                        $order = $parameters['order'];
+                        if (!is_array($order) || count($order) !== 2 || (strtoupper($order[1]) != 'ASC' && strtoupper($order[1]) != 'DESC')) {
+                            throw new \Exception('Order filter must have two parameters with second once asc or desc');
+                        }
+                        if ($order[0] === 'id') {
+                            $order[0] = 'uuid';
+                        }
+                        $queryStr .= ' ORDER BY S.'.$order[0].' '.$order[1];
+                    }
+
+                    $query = $this->dm->createPhpcrQuery($queryStr, QueryInterface::JCR_SQL2);
+
+                    if (isset($parameters['limit'])) {
+                        $query->setLimit($parameters['limit']);
+                    }
+                    if (isset($parameters['start'])) {
+                        $query->setOffset($parameters['start']);
+                    }
```

Comment:
We try to keep code standards close to Symfony (http://symfony.com/doc/current/contributing/code/standards.html), and there is always new line after if block (as long as it's not the last one thing in function). In rest of file there is new line after if block - why here we should skip it?